### ISSUE TITLE
Update docs content and include that in parsetoml.nim

### ISSUE THREAD

**Issue by kaushalmodi** [NimParsers/parsetoml#27]
That way http://nimparsers.github.io/parsetoml/ will contain all that info too.

Then remove https://parsetoml.readthedocs.io/en/latest/. At the moment more than half of its content is deprecated.

**Comment by kaushalmodi** [comment]
@PMunch As I don't know much of the implementation details of parsetoml, would you be able to update https://github.com/NimParsers/parsetoml/blob/master/docs/introduction.rst? 

I deleted rest of the files in https://github.com/NimParsers/parsetoml/commit/378593b67943208b574a99e9a0b5823366944ef5 as they were all outdated.

**Comment by PMunch** [comment]
Yes, I've been intending to write some proper documentation for this. But I was hesitant of the new JSON-esque interface. So I didn't want to write some documentation just to re-implement it and have to re-write the all of the documentation. But right now the docs are just horrible, so I'll write something proper.
